api/nakadi-event-bus-api.yaml

```diff
@@ -1246,7 +1254,16 @@ definitions:
     properties:
       name:
         type: string
-        description: Name of the strategy.
+        enum: ['hash', 'user_defined', 'random']
```

Comment:
I would ratter keep the documentation in the API yml file only. Since it's more convenient for the users to understand Nakadi capabilities without been forced to run some GET in the command line.